File: utils/VectorEnginePlugin.ref/pares/funciones/primSetClippingSpec.c
/* VectorEnginePlugin>>#clippingSpec: */
EXPORT(sqInt)
primSetClippingSpec(void)
{
	int *anIntegerArray;

	if (!(isWords(stackValue(0)))) {
		return primitiveFailFor(PrimErrBadArgument);
	}
	anIntegerArray = firstIndexableField(stackValue(0));
	clippingSpec = anIntegerArray;
	if (!(failed())) {
		pop(1);
	}
	return null;
}

File: utils/VectorEnginePlugin.ref/pares/funciones/primSetClippingSpec.js
function primSetClippingSpec(argCount) {
    if (!isWords(stackValue(0))) {
        primitiveFailFor(PrimErrBadArgument); return false;
    }
    clippingSpec = int32Of(stackValue(0));
    if (!failed()) pop(1);
    return !failed();
}
